haskell/stack.yaml

```
@@ -30,7 +30,7 @@ extra-deps:
     https://gitlab.com/morley-framework/morley-metadata.git
     # ^ CI cannot use ssh, so we use http clone here
   commit:
-    abc385928e69b6429595c6169b2494165aae9c77 # master
+    eb8511f671b3ffc9ec098fc827e251a4537c5511 # master
```

The `master` branch has gone forward in the last few days, I think we should bump this dependency to the latest version:
```suggestion
    7b3ee35067f368e627029d71f3b5453b430bac0d # master
```

Note: it will however require some changes in `MichelsonStorageView` usages.